.github/workflows/test.yaml

```diff
@@ -0,0 +1,15 @@
+name: test
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
```

Comment:
Your Dockerfile is using node 14, do you want to upgrade the version here?

Reply:
yeah I'll change it here too, thanks for noticing that.